public/js/src/module/offline-app-worker-partial.js

```diff
@@ -53,7 +51,8 @@ self.addEventListener( 'fetch', event => {
 
                 // TODO: we have a fallback page we could serve when offline, but tbc if that is actually useful at all
 
-                return fetch( event.request, { credentials: 'same-origin' } )
+                // To bypass any HTTP caching, always obtain resource from network
+                return fetch( event.request, { credentials: 'same-origin', cache: 'reload' } )
```

Comment:
Makes sense: if the service worker is explicitly managing its own cache, there's no need for another layer of opaque caching on top of that